demo_umami_content/tests/src/Functional/UninstallDefaultContentTest.php

```diff
@@ -46,6 +46,20 @@ public function testReinstall() {
     $this->assertEquals(0, $count);
```

Comment:
We're testing its here on re-install, but we're not testing its there on install (ie above line 34).

I suspect it might not be (based on my comment above)

Reply:
and when I say 'install' I mean 'of the profile by the ::setup method'

Reply:
@larowlan Line 51 tests the presence of block on /recipe page. Then module is uninstalled again tested for no presence and then re-installed and tested for presence on line 61